packages/x-components/src/x-modules/scroll/components/scroll.vue

```diff
@@ -60,23 +62,14 @@
     @Prop({ default: 'scrollId' })
     public id!: string;
 
-    /**
-     * If true (default), sets the scroll position of the {@link BaseScroll} component to top when
-     * an {@link XEventsTypes.UserAcceptedAQuery} event is emitted.
-     *
-     * @public
-     */
-    @Prop({ default: true })
-    protected resetOnQueryChange!: boolean;
```

Comment:
What happened with this prop? If we're going to remove it please remove it also in the `BaseScroll` and `Scroll` doc examples

Reply:
Remove it in the BaseScroll docs also please.